_integration-test/run.sh

```diff
@@ -171,6 +171,28 @@ $dcr --no-deps web python3 /etc/sentry/test-custom-ca-roots.py
 source _integration-test/custom-ca-roots/teardown.sh
 echo "${_endgroup}"
 
+echo "${_group}Test that replays work ..."
+echo "Creating test replay..."
+TEST_REPLAY_ID=$(
+  export LC_ALL=C
+  head /dev/urandom | tr -dc "a-f0-9" | head -c 32
+)
+TIME_IN_SECONDS=$(date +%s)
+curl -sf --data '{"event_id":"'"$TEST_REPLAY_ID"'","sdk":{"name":"sentry.javascript.browser","version":"7.38.0"}}
```

Comment:
for some reason the bash syntax of `$'` for C escaped strings for newlines wasn't working so just put in the real values